terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/ExistInCodeList.java

```diff
@@ -31,6 +33,8 @@
 import org.terasoluna.gfw.common.codelist.validator.ExistInCodeListValidatorForCharacter;
 import org.terasoluna.gfw.common.codelist.validator.ExistInCodeListValidatorForCharSequence;
 
+import org.terasoluna.gfw.common.codelist.ExistInCodeList.List;
```

Comment:
- [ ] Why need import inner class `List`?

Reply:
`@Repeatable` annotation needs inner class `List`. For example, [Bean Validation 2.0.1 @NotNull annotation](https://www.google.co.jp/search?source=hp&ei=4dpPXLjlJILy8AW-lqK4DA&q=bean+validation+github&btnK=Google+%E6%A4%9C%E7%B4%A2&oq=bean+validation+github&gs_l=psy-ab.3..0i19j0i8i30i19l2.2542.6863..7403...0.0..0.91.1224.16......0....1..gws-wiz.....0..0i131j0i4j0j0i131i4j0i30j0i4i30.TwSw5kuq8GI) import inner class `List` to use `@Repeatable`.

Reply:
That's not the reason.
Inner classes are defined within classes, so import is unnecessary within class.
This case is using inner class outer this class, so import is necessary.
Or `@Repeatable(ExistInCodeList.List.class)` is correct too.
(self-solving)